storage/range.go

```diff
@@ -614,7 +613,13 @@ func (r *Range) addReadOnlyCmd(ctx context.Context, args proto.Request, reply pr
 		if header.Timestamp.Equal(proto.ZeroTimestamp) {
 			header.Timestamp = r.rm.Clock().Now()
 		}
-		intents, err := r.executeCmd(r.rm.Engine(), nil, args, reply)
+		executedReply, intents, err := r.executeCmd(r.rm.Engine(), nil, args)
+		if executedReply.Header().Error != nil {
+			panic("the world is on fire")
```

Comment:
Yeah, this message sucks, but it's pretty transient as I move this code up the stack.